### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nikita-fuchs**
add a few more information to get people going more easily.

### CODE REVIEW

**Review by dincho — DISMISSED** [review]

**Review by hanssv — DISMISSED** [review]

**Review by uwiger — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -48,10 +48,17 @@ The `master` branch tracks the ongoing efforts towards the next stable release t
...
+
 By using the installer to install the latest stable version:
 ```bash
 bash <(curl -s https://install.aeternity.io/install.sh)
 ```
+See the documentation for [starting](https://docs.aeternity.io/en/stable/operation/#start-the-node) and [configuring](https://docs.aeternity.io/en/stable/configuration/#example) the node.
+
+
+
+**Linux / Mac / Win**
```
Comment (velzevur) [inline README.md:61]: Win?

**Review by velzevur — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nikita-fuchs) [reply README.md:61]: oof good catch @velzevur , let me find someone to test on windows...

**Review by uwiger — DISMISSED** [review]

**Review by dincho — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -48,19 +48,34 @@ The `master` branch tracks the ongoing efforts towards the next stable release t
...
+$ docker pull aeternity/aeternity
+$ docker run -p 3013:3013 -p 3015:3015 \
     -v ~/.aeternity/maindb:/home/aeternity/node/data/mnesia \
     aeternity/aeternity
 ```
+**Windows**
+```bash
+$ mkdir %APPDATA%\aeternity\maindb
+$ docker pull aeternity/aeternity
+$ docker run -p 3013:3013 -p 3015:3015 -v %APPDATA%/aeternity/maindb:/home/aeternity/node/data/mnesia aeternity/aeternity
```
Comment (dincho) [inline README.md:77]: please make it multi-line to be consistent with the above command

Comment (nikita-fuchs) [reply README.md]: no, because in command prompt and powershell the newline escape sign is inconsistent, we'll keep it simple.

**Review by dincho — APPROVED** [review]

**Review by uwiger — APPROVED** [review]

**Review by velzevur — APPROVED** [review]

**Pull request merged**